client/src/redux/ga-saga.test.js

```diff
@@ -11,8 +11,8 @@ describe('ga-saga', () => {
     const mockEventPayload = {
       type: 'event',
       data: {
-        category: 'Donation',
-        action: 'year end gift paypal button click'
+        category: 'Map Challenge Click',
+        action: '/learn'
```

Comment:
Changed the mock data to reduce the debugging time before the feature is complete.